theme_test_custo/static/src/scss/primary_variables.scss

```diff
@@ -0,0 +1,108 @@
+// -------------------------------------------------------------------------- //
+// PRESETS
+// -------------------------------------------------------------------------- //
+
+$o-website-values-palettes: (
+    (
+        // Colors
+        'color-palettes-name':              'custom-theme',
+
+        // Fonts
+        'font':                             'Source Sans Pro',
+        'headings-font':                    'Poppins',
+        'navbar-font':                      'Source Sans Pro',
+        'buttons-font':                     'Source Sans Pro',
+
+        // Header
+        'header-template':                  'custom-theme',
+        'logo-height':                      4rem,
+
+        // Footer
+        'footer-template':                  'custom-theme',
```

Comment:
I keep repeating not to add 500 spaces to aligns the values in maps, maybe let's not do it here either to prevent this to be copy/pasted elsewhere? :)